grapi/backend/kopano/attachment.py

```diff
@@ -191,6 +191,26 @@ def on_get_by_id(self, req, resp, itemid, attachmentid=None):
         item = get_item(req, itemid)
         self._response_attachments(req, resp, item, attachmentid)
 
+    def on_get_embedded_item_by_id(self, req, resp, itemid, attachmentid):
+        """Response attachment embedded item by itemid with attachment ID.
+
+         Args:
+            req (Request): Falcon request object.
+            resp (Response): Falcon response object.
+            item (Item): instance of an item.
+            attachmentid (str): attachment ID which is related to the item. Must be set.
```

Comment:
```suggestion
            attachmentid (str): attachment ID which is related to the item.
```
As it doesn't have any default value, it's not needed to mention it. Because Python will raise an error anyway.